doc/advanced/content/conf.py

```diff
@@ -126,4 +126,4 @@
 file_insertion_enabled = True
 raw_enabled = True
 # Set up doxylink
-doxylink = {'pcl' : ('../../../build/doc/doxygen/pcl.tag', 'http://docs.pointclouds.org/trunk/')}
+doxylink = {'pcl' : ('../../../s/doc/doxygen/pcl.tag', 'https://pointclouds.org/documentation/')}
```

Comment:
/s/ is a CI specific path?

Reply:
In a way. `pcl.tag` is written into the directory structure of the source (`doc/doxygen`, next to `doxyfile.in`). I tried to make the location where `pcl.tag` is written to configurable, but couldn't find a solution that works on the CI (I removed those experimental commits again). I am also not sure how the previous version with `.../build/...` could have worked, because, as mentioned, `pcl.tag` is written to the source directory, not to any build directory.